reviews/utils/review_utils.py

```diff
@@ -61,6 +62,14 @@ def start_supervisor_phase(proposal):
     msg_html = render_to_string('mail/concept_creator.html', params)
     send_mail(subject, msg_plain, settings.EMAIL_FROM, [proposal.created_by.email], html_message=msg_html)
 
+    if proposal.other_applicants:
+        params['creator'] = proposal.created_by.get_full_name()
+        msg_plain = render_to_string('mail/concept_other_applicants.txt', params)
+        msg_html = render_to_string('mail/concept_other_applicants.html', params)
+        other_applicants_emails = [applicant.email for applicant in proposal.applicants.all()]
+        other_applicants_emails.remove(proposal.created_by.email)
```

Comment:
You probably also want to remove the supervisor, if it's in there.

It's very common that the supervisor is also included as an applicant.